feat: update browserslist version

yukipastelcat opened the pull request:
# Pull request summary

Related to [AUT-2510](https://oat-sa.atlassian.net/browse/AUT-2510).

Updated @oat-sa/browserslist-config-tao version to 1.0.0 in order to include removal of IE support

# How to test

1. Run unit tests via `npm test`

# Checklist

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not master

yukipastelcat committed:
feat: update browserslist version

- Updated browserslist-config-tao version to 1.0.0 in order to include removal of IE support

oatymart reviewed (approved):
Tested locally, the application still works after this is bundled. I haven't tested more exotic browsers than the latest Chrome & Safari.

- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

yukipastelcat committed:
chore: update dependencies

- Updated oat-sa/browserslist-config-tao
- Updated oat-sa/tao-core-sdk
- Updated oat-sa/tao-core-ui
